pkg/catalog/catalog_test.go

```diff
@@ -83,4 +82,19 @@ var _ = Describe("Catalog", func() {
 			})
 		})
 	})
+
+	Describe("GetGreaterThan", func() {
+		It("lists all available versions which are greater than the current version", func() {
```

Comment:
nit: lets call out in the code & test its an ordered list, to make sure folks know :smile: